gem_city.rb

```diff
@@ -0,0 +1,50 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+  def initialize
+    @people = {
+      thieves: 5,
+      officers: 1 }
```

Comment:
It would be simpler if these weren't stuck together in `@people`. Try defining them as `@thieves` and `@officers`.